webservices/resources/dates.py

```diff
@@ -60,24 +58,25 @@ def get(self, **kwargs):
 @doc(description='FEC election dates since 1995.')
 class ElectionDatesView(DatesResource):
 
-    model = models.ElectionDates
+    model = models.ElectionDate
+
     @property
     def date_column(self):
         return self.model.election_date
 
-    filter_fields = {
-        'election_state',
-        'election_district',
-        'election_party',
-        'office_sought',
-        'election_date',
-        'trc_election_type_id',
-        'trc_election_status_id',
-        'update_date',
-        'create_date',
-        'election_year',
-        'pg_date',
-    }
+    filter_multi_fields = [
```

Comment:
In that case, what do you think about giving the filter and model field clearer names? Also, I'm still missing what primary/general date means--is it the date of the primary election or the general election? Also, if `trc_election_type_id` and `trc_election_status_id` aren't going to be meaningful to users, should we lose these filters? Should those columns be included in the serialized output?